packages/web/src/app/members/page.tsx

```diff
@@ -0,0 +1,71 @@
+interface MemberInfo {
+  rank: string;
+  benefits: string[];
+  requirements?: string[];
+}
+
+const membershipLevels: MemberInfo[] = [
+  {
+    rank: "Free Player",
+    benefits: [
+      "Access to basic game features",
+      "Play against other players",
+      "Basic orbs and powers",
+    ],
+  },
+  {
+    rank: "Premium Member",
+    benefits: [
+      "Access to all orbs and powers",
+      "Create private matches",
+      "Custom player profile",
+      "Ad-free experience",
```

Comment:
I'd omit this line, since the ads don't load at this point (I think?) and anyway I would prefer people not to think that we're profiting of those